README.rst

```diff
@@ -181,11 +181,18 @@ DNS lookup...
 * Browser checks if the domain is in its cache.
 * If not found, calls ``gethostbyname`` library function (varies by OS) to do
   the lookup.
+* ``gethostbyname`` will try to resolve the lookup against the system
+  cache (nscd, nslcd)
+* If not found, ``gethostbyname`` will obay the configuration of
+  ``/etc/nsswitch.conf`` or Windows equivalent (Service Providers in
+  registry) and figure out who is going to query next, assuming the order is
```

Comment:
The wording on 188 and 189 is strange, and I don't think is saying what you want to say.